boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/JDBCBoosterPackConfigurator.java

```diff
@@ -9,6 +9,8 @@
 public class JDBCBoosterPackConfigurator implements BoosterPackConfigurator {
 
     private String jdbcDefault = "jdbc-4.1";
+    
+    String featureGAV = null;
```

Comment:
Should remove System.out.println() statements throughout.

Reply:
yup, i will hunt them down.